packages/malloy-vscode/package.json

```diff
@@ -175,19 +175,22 @@
       }
     ],
     "views": {
-      "malloy": [
+      "explorer": [
         {
           "id": "malloyHelp",
           "name": "Help",
-          "type": "webview"
+          "type": "webview",
+          "when": "resourceFilename =~ /malloy/"
```

Comment:
I'd recommend `"resourceFilename =~ /\\.malloy$/"` to avoid false positives like `malloy.txt`

Reply:
👍